### PR TITLE
README: update to current codebase

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ cat > /etc/pcw.ini << EOT
 # Your base url used to create links in email notifications. If it isn't
 # present, the first entry of settings.ALLOWED_HOSTS is used.
 base-url = https://publiccloud.qa.suse.de
+# Fallback list when namespaces request for certain feature but not defined there
+namespaces = VAULT_NAMESPACE, VAULT_NAMESPACE_2, ...
+# boolean value controling wether pcw is running in "harmless" testing mode.
+# if True pcw will not do any changes in target namespaces but just declare in log
+# message what will be done. Default value is False
+dry_run = [True,False]
```
Comment (grisu48) [inline README.md:39]: This is nice!

**Review by grisu48 — APPROVED** [review]
LGTM :rocket:

File: README.md [hunk 2]
```diff
@@ -3,9 +3,9 @@
 
 # OpenQA Public cloud Helper
 
-A web app which monitor, display and delete CSPs. It try to identify left over
+A web app which monitor, display and delete CSPs and linked resources. It try to identify left over
```
Comment (jlausuch) [inline README.md:6]:
```suggestion
A web app which monitors, displays and deletes CSPs and linked resources. It tries to identify left overs
```

File: README.md [hunk 3]
```diff
@@ -3,9 +3,9 @@
...
 instances and also perform some cleanup task. For instance it delete old custom
```
Comment (jlausuch) [inline README.md:7]:
```suggestion
instances and also performs some cleanup task. For instance it deletes old custom
```

File: README.md (hunk 2 above)
Comment (asmorodskyi) [reply README.md:6]: what I can say , I have grammar issues in my native language too :D . thanks !

File: README.md [hunk 4]
```diff
@@ -31,6 +31,12 @@ cat > /etc/pcw.ini << EOT
 # Your base url used to create links in email notifications. If it isn't
 # present, the first entry of settings.ALLOWED_HOSTS is used.
 base-url = https://publiccloud.qa.suse.de
+# Fallback list when namespaces request for certain feature but not defined there
```
Comment (jlausuch) [inline README.md:34]: can we separate the different options using blank line?

File: README.md [hunk 5]
```diff
@@ -31,6 +31,12 @@ cat > /etc/pcw.ini << EOT
...
+namespaces = VAULT_NAMESPACE, VAULT_NAMESPACE_2, ...
+# boolean value controling wether pcw is running in "harmless" testing mode.
```
Comment (jlausuch) [inline README.md:36]:
```suggestion
# Boolean value controlling wether pcw is running in "harmless" testing mode.
```

File: README.md [hunk 6]
```diff
@@ -40,15 +46,18 @@ namespaces = VAULT_NAMESPACE, VAULT_NAMESPACE_2, ...
 # Use this option only during development! When enabled last response from
 # vault is stored in a file 'auth.json'. The file is located in '/tmp/pcw'
 use-file-cache = False
+providers = csp1[, csp2]...
```
Comment (jlausuch) [inline README.md:49]: is this the right usage of `[]`?

File: README.md [hunk 7]
```diff
@@ -40,15 +46,18 @@ namespaces = VAULT_NAMESPACE, VAULT_NAMESPACE_2, ...
...
 
+# XXX is the name of a namespace.
```
Comment (jlausuch) [inline README.md:51]: minor thing, but I'd prefer using XYZ than XXX :p

File: README.md [hunk 8]
```diff
@@ -57,25 +66,42 @@ to = RECEIPE_ADDRESS1[, RECEIPE_ADDRESS2]
...
+# - notify/smtp-port
+# - notify/from
 [notify.namespace.XXX]
-# Optional section to set a specific receiver of left over notifications for
-# a defined vault namespace. XXX should be replaced with the vault namespace
 to = RECEIPE_ADDRESS_NS_1[, RECEIPE_ADDRESS_NS_2]
 
 
+# while namespaces defined in vault section would be scanned for
+# orphaned instances. This list of namespaces will be scanned for any
```
Comment (jlausuch) [inline README.md:81]: This sentence is missing something, maybe you want to use coma instead of dot?

File: README.md [hunk 9]
```diff
@@ -95,3 +121,15 @@ pytest -v --cov=./ --cov-report=html && $BROWSER htmlcov/index.html
...
 
+## Debug
+
+To simplify problem investigation pcw has two [django commands](https://docs.djangoproject.com/en/3.1/howto/custom-management-commands/) :
+
+[cleanup](ocw/management/commands/cleanup.py)
+
+[updaterun](ocw/management/commands/updaterun.py)
+
+this allow to trigger core functionality without web UI. It is highly recommended to use `dry_run = True` in `pcw.ini` in
```
Comment (jlausuch) [inline README.md:132]:
```suggestion
this allows triggering core functionality without web UI. It is highly recommended to use `dry_run = True` in `pcw.ini` in
```

**Review by jlausuch — APPROVED** [review]

**Pull request merged**